src/main/java/net/neoforged/moddevgradle/dsl/RunModel.java

```diff
@@ -140,7 +143,16 @@ public void jvmArgument(String arg) {
      *
      * @see ModModel
      */
-    public abstract SetProperty<ModModel> getMods();
+    public abstract SetProperty<ModModel> getLoadedMods();
```

Comment:
Maybe a better name exists, for example `localMods`?

Reply:
Well, it's not all the mods that are loaded by FML. It's only the `mods {}` to load. I suppose it works well enough with `loadedMods`.